README.md

```diff
@@ -1,12 +1,48 @@
 ![NodeSchool Montréal Logo](http://nodeschool.io/montreal/i/NodeSchoolMTL.svg)
 
-# Develop NodeSchoolMTL
+# NodeSchool Montreal Website
 
 We use [Jekyll](https://jekyllrb.com/) to build [http://nodeschool.io/montreal](http://nodeschool.io/montreal). If you want to run the site or develop it locally you'll have to install [Jekyll](https://github.com/jekyll/jekyll) and [Jekyll-Sitemap](https://github.com/jekyll/jekyll-sitemap).
 
-`gem install jekyll`
+Aditionally, in order to suport both French and English
+content, we need to use gems (Ruby modules) that are not
+supported by GitHub's gh-pages Jekyll scripts. For this
+reason we run jekyll with the help of [Travis](https://travis-ci.com/).
 
-`gem install jekyll-sitemap`
+Notes to Mentors
+----------------
+
+If you are adding yourself or removing yourself as a 
+mentor, you should create a PR request, first having modified
+the following resources, as appropiate:
+
+   - _config.yml
+   - i/mentors/
+
+Generating the Site
+-------------------
+
+One you have made your changes, you can test them, by
+running (tested on macOS and Linux):
+
+`script/cibuild`
+
+`bundle exec jekyll serve`
+
+Note, that the `script/deploy` is intended to be used
+by Travis and should not be run locally. When run it
+will do all the magic necessary to take the generated
+site and push it to the gh-pages branch.
+
+Pull Request
+------------
+
+Remember when making your pull request (PR) to do it against
+the [gh-pages-src](https://github.com/nodeschool/montreal/tree/gh-pages-src) branch. Doing anything else will most definitely
```

Comment:
Why do we have other branches? Can we just have the one, or at the least set it as the default?

Reply:
Not sure why we have both `master` and `gh-pages`? As for `gh-pages-src`, this is where we are publishing from to `gh-pages`, as part of the Travis CI process. I wanted to do the changes incrementally, until I had feedback from the relevant people.

We could change things up, such that `master` and `gh-pages-src` are merged into one branch and that we publish to `gh-pages`. I don't want to do this until I have an okay from @pcuci 

GitHub does not support i18n at this point. There is the following issue open tracking this: https://github.com/github/pages-gem/issues/401

Reply:
Then let's remove `master`, or, better, remove `gh-pages-src` and rename it to `master`, and just have people push all commits against master, which would then generate `gh-pages` automatically. I think this is a more common flow. @pcuci sound good?

Reply:
@ajmas Can we go with this flow, then?

Reply:
Needs to be updated.